build/patches/Move-some-account-settings-back-to-privacy-settings.patch

```diff
@@ -142,8 +147,7 @@ diff --git a/chrome/android/java/src/org/chromium/chrome/browser/privacy/setting
 +     *  will the AA switch be assigned a value).
 +     */
 +    private boolean shouldShowAutofillAssistantPreference() {
-+        return mSharedPreferencesManager.contains(
-+                        ChromePreferenceKeys.AUTOFILL_ASSISTANT_ENABLED);
++        return true;
```

Comment:
in my opinion it can become `return AssistantFeatures.AUTOFILL_ASSISTANT.isEnabled();`
see [my version](https://github.com/uazo/bromite/blob/next/v99/build/patches/Move-some-account-settings-back-to-privacy-settings.patch)

Reply:
I have fixed this but did not introduce the other 2 functions as in your version.

Reply:
of course, no problem, the meaning is the same, then, when you publish it, I take your changes

Reply:
I committed the changes which address all the resolved feedback.